### ISSUE TITLE
Homebrew installation instructions do not work

### ISSUE THREAD

**Issue by carlosefr** [hashicorp/vagrant#12354]
The instructions at https://www.vagrantup.com/downloads to install vagrant on macOS via homebrew do not work:
```
brew tap hashicorp/tap
brew install hashicorp/tap/vagrant
```

The tap is correctly registered with brew, but the second command errors out:
```
==> Searching for similarly named formulae...
Error: No similarly named formulae found.
Error: No available formula or cask with the name "hashicorp/tap/vagrant".
==> Searching for a previously deleted formula (in the last month)...
Error: No previously deleted formula found.
```

### Host operating system

macOS 10.15.7 (Catalina)

**Comment by lkysow** [comment]
From https://github.com/hashicorp/homebrew-tap#what-packages-are-available

> Note: Vagrant is available as a cask from Homebrew core and not provided by this tap.

So you should use `brew install vagrant`. But also the downloads page needs to be fixed.

**Comment by chrisroberts** [comment]
Closing as fixed via #12358
